Exercise_A/index.html

```diff
@@ -4,6 +4,22 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+      crossorigin="anonymous"/>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"

```

Comment:
Conventionally you should be adding this script tag and the ones below, within the body, before the closing tag.